modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -71,7 +71,7 @@ The ID determines which image the installation program must download.
 +
 [source,terminal]
 ----
-$ export RHCOS_QEMU_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/coreos/rhcos.json  | jq .images.qemu.path | sed 's/"//g')
+$ export RHCOS_QEMU_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/coreos/rhcos.json  | jq '.architectures.x86_64.artifacts.qemu.formats["qcow2.gz"].disk.location' | sed 's/"//g')
```

Comment:
You can drop the `sed` part of this oneliner if you use `jq -r`

For example:

```bash
$ curl -s -S https://raw.githubusercontent.com/openshift/installer/604a5a16e76392b01bf67971ac75d6748b07bcbd/data/data/coreos/rhcos.json | jq '.architectures.x86_64.artifacts.qemu.formats["qcow2.gz"].disk.location' | sed 's/"//g'
https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.10/410.84.202201251210-0/x86_64/rhcos-410.84.202201251210-0-qemu.x86_64.qcow2.gz

$ curl -s -S https://raw.githubusercontent.com/openshift/installer/604a5a16e76392b01bf67971ac75d6748b07bcbd/data/data/coreos/rhcos.json | jq -r '.architectures.x86_64.artifacts.qemu.formats["qcow2.gz"].disk.location'
https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.10/410.84.202201251210-0/x86_64/rhcos-410.84.202201251210-0-qemu.x86_64.qcow2.gz
```

Reply:
oh awesome, thanks! I've updated the doc to reflect this